cookbook/custom-error-page.rst

```diff
@@ -0,0 +1,95 @@
+Custom error page
+=================
+
+With Sulu it is very easy to customize the error pages for you website users.
+You can define for each http-status-code an own template.
+
+Configuration
+-------------
+
+The following code-block shows a default configuration for the exception
+templates. If you want to add an own exception for example 400 you can simply
+add it to the list. You can specify that for each webspace.
+
+.. code-block:: xml
+
+    <theme>
+       <key>default</key>
+
+        <error-templates>
+            <error-template default="true">ClientWebsiteBundle:views:error.html.twig</error-template>
+            <error-template code="404">ClientWebsiteBundle:views:error404.html.twig</error-template>
+            <error-template code="500">ClientWebsiteBundle:views:error500.html.twig</error-template>
+        </error-templates>
+    </theme>
+
+The `ExceptionController` uses the status-code of the response to determine
+which template is responsible for the exception. If no special template is
+defined it uses the `default` template.
+
+Twig-Template
+-------------
+
+In the twig-template you can use your website master template to reuse your
+style.
+
+.. code-block:: html
+
+	{% extends "ClientWebsiteBundle:views:master.html.twig" %}
+
+	{% block title %}Error {{ status_code }}{% endblock %}
+
+	{% block content %}
+	    <h1>Error {{ status_code }}</h1>
+	    <p>{{ status_text }}</p>
+
+	    <p>{{ exception.message }}</p>
+	{% endblock %}
+
+.. note::
+
+    Be carefully which variable you use in your `master.html.twig`.
```

Comment:
`Be careful ...`
And please also add why it is important. I also guess it is the other way round, in `master.html.twig` you are allowed to use everything, but you are not allowed to use variables in the block defined in the master template since this will break the preview. Or are you talking about something different?